content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -5,9 +5,16 @@ metaDescription: Prisma Client features that are currently in preview.
 
 <TopBlock>
 
-The following [Preview](/about/prisma/releases#preview) feature flags are available for Prisma Client and the Prisma schema:
+When we release a new Prisma Client or schema feature, it often starts in preview so that you can test it and submit your feedback. After we  improve the feature with your feedback and are satisfied with the internal test results, we promote the feature to generally availability.
```

Comment:
```suggestion
When we release a new Prisma Client or schema feature, it often starts in preview so that you can test it and submit your feedback. After we improve the feature with your feedback and are satisfied with the internal test results, we promote the feature to general availability.
```